app.py

```diff
@@ -68,8 +71,36 @@ def save():
     gt.save_and_return()
     return "a"
 
+##########################################
+# for test purposes
+# basic_science to ganesha_statue
+
+
+def test_stretch_1():
+    start = tc.basic_science.point_name
+    end = tc.ganesha_statue.point_name
+    gt.add_tree(tc.tree1.point_coordinate, start, end)
+    gt.add_tree(tc.tree2.point_coordinate, start, end)
+    gt.add_tree(tc.tree3.point_coordinate, start, end)
+    gt.save_and_return()
+
+
+# ganesha_statue to hostel_turn
+def test_stretch_2():
+    start = tc.ganesha_statue.point_name
+    end = tc.hostel_turn.point_name
+    gt.add_tree(tc.tree4.point_coordinate, start, end)
+    gt.add_tree(tc.tree5.point_coordinate, start, end)
+    gt.add_tree(tc.tree6.point_coordinate, start, end)
+    gt.save_and_return()
+
+
+test_stretch_1()
+test_stretch_2()
+##########################################
+
```

Comment:
Test Case Functions for each stretch 
`basic_science to ganesha_statue` , `ganesha_statue to hostel_turn`

Utilised test_coordinates for test nodes.